### ISSUE TITLE
Microsoft Edge can't uninstall 

### ISSUE THREAD

**Issue by Alcycber** [ChrisTitusTech/winutil#2672]
Trying to uninstall Microsoft edge using win utility keeping getting a error saying ====Chris Titus Tech=====
=====Windows Toolbox=====
Running Script for WPFTweaksRemoveEdge
[] Registry key not found: HKLM:\SOFTWARE\WOW6432Node\Microsoft\EdgeUpdate\ClientState\{56EB18F8-B008-4CBD-B6D2-8C97FE7E9062}
Cannot find uninstall methods for
=================================
--     Tweaks are Finished    ---
=================================

**Comment by crazycheetah42** [comment]
I have also noticed that Edge won't uninstall.

**Comment by astroprogs11** [comment]
Yup, same issue.

**Comment by ChrisTitusTech** [comment]
Looks like Microsoft patched the region workaround and it no longer removes Edge... FML.

**Comment by Alcycber** [comment]
@ChrisTitusTech Thank you so much for replying it was driving me mad 😂 got any suggestions on remember removing?.

**Comment by UltraCakeBakery** [comment]
Can't you just swap the edge browser with chrome? As in, literally swap the binary files and change the icon for every shortcut? Or maybe symlink some shit? Idk man but there must be a creative way to just make windows open chrome for everything and then just hide edge from the user?

**Comment by crazycheetah42** [comment]
@ChrisTitusTech I believe there is a NoRemove registry key for Edge. That makes it uninstallable from the regular Settings app. Maybe we could swap the "Remove Edge" button for a "Make Edge uninstallable" button.

**Comment by crazycheetah42** [comment]
Yeah, the path is HKLM\SOFTWARE\WOW6432NODE\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft Edge

**Comment by Linget0** [comment]
We shouldn't even need to change the region to unlock the uninstall, the best way is if the script could gain permission to modify the file, go into the ServicesRegionPolicy json and edit the text to change "Edge is uninstallable" default state from disabled to enabled then restore the ACL. Then the only other thing would be to remove the "NoRemove" registry key that grays out the uninstall. Heres a video for concept https://www.youtube.com/watch?v=_utrsNxQiR0

**Comment by 4yman-0** [comment]
[Kihpla](https://github.com/Kihpla) from (#2735 Edge Uninstall) linked to a [UninstallEdge.ps1](https://gist.github.com/ave9858/c3451d9f452389ac7607c99d45edecc6) file to uninstall Edge.

**Comment by 4yman-0** [comment]
This is a duplicate of (#2544 Removing Edge and Microsoft apps doesn't work).

**Comment by ToddIndry37** [comment]
> Yeah, the path is HKLM\SOFTWARE\WOW6432NODE\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft Edge

@crazycheetah42 Yeah it did, but unfortunately I think Microsoft patched it to block the Edge uninstall process.

**Comment by xd003** [comment]
https://github.com/ShadowWhisperer/Remove-MS-Edge 
This works perfectly fine for Uninstalling Edge if that helps in any way
